examples/gcp_cli.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demo CLI tool for GCP."""
+
+import argparse
+from libcloudforensics import gcp
+
+
+def ListInstances(args):
+  """List GCE instances in GCP project.
+
+  Args:
+    args (dict): Arguments from ArgumentParser.
+  """
+
+  project = gcp.GoogleCloudProject(args.project)
+  instances = project.ListInstances()
+
+  print('Instances found:')
+  print(instances)
```

Comment:
This will just print a blob of instance objects. It would be nice to iterate over instances and print their `name`?